RSSReader/View/FeedViewController/FeedViewController.m

```diff
@@ -25,6 +25,9 @@ @interface FeedViewController () <UITableViewDataSource, UITableViewDelegate, Cu
 NSString * const kErrorTitle = @"Error";
 NSString * const kErrorRetry = @"Retry";
 NSString * const kCellID = @"cellId";
+NSString * const kReloadAlertMessage = @"Reload feed?";
+NSString * const kYesActionTitle = @"Yes";
```

Comment:
I would better use localised string keys for this.

Reply:
This constants are keys in NSLocalizedStrings

Reply:
Good.